music_album.rb

```diff
@@ -0,0 +1,20 @@
+require_relative 'item'
+
+class MusicAlbum < Item
+  attr_accessor :on_spotify
+
+  def initialize(on_spotify)
+    super(params)
```

Comment:
[Optional]You did a nice job Pablo 👍  It would be better to call super() in the child file without a parameter if you do have not something to put inside (leave it empty)

Reply:
Yes u right !
I did it know 
@francksefu